.changeset/four-humans-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': minor
+---
+
+Installation for stylelint-use-logical CSS property rule, converted all physical properties in .SCSS files to their logical equivalent.
```

Comment:
changeset notes should be user-facing. stylelint and sass are irrelevant development-only details.

```suggestion
---
'@itwin/itwinui-css': minor
'@itwin/itwinui-react': minor
---

Converted all physical CSS properties to their logical equivalents.
```